front/src/utils/Utils.js

```diff
@@ -209,7 +209,7 @@ export async function validateLogin() {
 // on success true
 // else returns string trying to explain what broke
 // requires reservation and schedule to exist
-export async function rangeSupervision(rsId, srsId, rangeStatus, rsScheduled) {
+export async function rangeSupervision(rsId, srsId, rangeStatus, rsScheduled, token, supervisor) {
```

Comment:
Token looks like an unused variable here

Reply:
it's used in something as removing it results in the following error:
Error: Undefined binding(s) detected when compiling SELECT. Undefined column(s): [user.id] query: select "user"."email" from "user" where "user"."id" = ?

Reply:
Interesting... Well, we can let this be there if removing it causes this slightly random error to come up and debugging it takes too long.

Reply:
I think removing this.state.token from variables given to rangeSupervision at src/scheduling/Scheduling.js row 650 fixes this

Reply:
done